nemesis/menu.py

```diff
@@ -269,7 +269,7 @@ def next_ap_needed(checkpoint):
     if checkpoint == 25:
         return 22
     if checkpoint == 26:
-        return 42
+        return 43
```

Comment:
This is why I don't like magic numbers. I have no idea if this change is correct or not

Reply:
This isn't magic, it's just the number of levels we need before we can do each of the sphere grid sections, defined later in the file.